airflow/cli/commands/celery_command.py

```diff
@@ -96,6 +99,31 @@ def _serve_logs(skip_serve_logs: bool = False):
         sub_proc.terminate()
 
 
+@after_setup_logger.connect()
+def logger_setup_handler(logger, **kwargs):
+    # Setup levels at which logs go to stderr and stdout if required
+    if conf.has_option("logging", "reset_and_split_logging") and conf.get(
+        "logging", "reset_and_split_logging"
+    ):
```

Comment:
I think you can do it in one call like this which should be the same
```
if conf.get("logging", "reset_and_split_logging", fallback=None):
```

Reply:
Updated.